projects/js-toolkit/packages/dev-server/bin/liferay-dev-server.js

```diff
@@ -5,7 +5,7 @@
  * SPDX-License-Identifier: LGPL-3.0-or-later
  */
 
-import server from '../lib/server';
+const {default: server} = require('../lib/server');
```

Comment:
Ha, good catch!

I forgot I was testing this with `NODE_OPTIONS='--experimental-specifier-resolution=node'` just for fun

![hallelluja](https://user-images.githubusercontent.com/905006/137104338-cb19d766-9760-4f71-8e76-462c2a57f3f6.gif)